doc/releases/release-notes-2.4.rst

```diff
@@ -82,6 +82,9 @@ API Changes
   const qualifier loss on ISRs, all ISRs now take a ``const *void`` as a
   paremeter as well.
 
+* The ``_gatt_``/``_GATT_`` infix for HRS, DIS and BAS was removed for the API
```

Comment:
```suggestion
* The `_gatt_` and `_GATT_` infixes have been removed for the HRS, DIS and BAS APIs and Kconfig functions.
```

Reply:
Applied your suggestion. I was a bit unsure of `Kconfig functions` was the right terminology (rather than `Kconfig options`, so I went with your suggestion. Please let me know if `options` is the better choice.